_data/navbars/release-notes.yml

```diff
@@ -1,6 +1,6 @@
-title: Release notes
+title: Release Notes
```

Comment:
This should stay as "Release notes" - we use sentence casing for all of our titles, etc.